src/trend/recommendations.md

```diff
@@ -224,7 +224,7 @@ begin
     &   " with quantity " + %string(restockReq.Quantity))
 end
 ```
-this loop iterates over the `restockRequests` ArrayList. For each restock request, it retrieves the corresponding inventory item to display the item's name and the recommended restock quantity. This is a very simple way to output the recommendations but at least we can see that the subroutine is working. Let's build and run the program again.
+This loop iterates over the `restockRequests` array list. For each restock request, it retrieves the corresponding inventory item to display the item's name and the recommended restock quantity. This is a very simple way to output the recommendations, but at least we can see that the subroutine is working. Let's build and run the program again.
```

Comment:
ArrayList is the type of the variable that we're iterating over, I dont think i want to split that into 'array list'